### PR TITLE
Cypress E2E for examples

### PR DESCRIPTION
**Pull request by Zeko369**
### Pull Request Type

Setup and add Cypress tests to example/store

- [x] Feature

### Checklist

- [x] Tests added for changes

### What's the reason for the change? :question:

Closes: #97

### What are the changes and their implications? :gear:

- [x] Setup Cypress
- [x] Update eslint to not yell at us for using global stuff
- [x] Write cypress tests
	- [x] /
    - [x] /products
    - [x] /products/ssr
    - [x] /products/[handle]
    - [x] /admin
	- [x] /admin/products
	- [x] /admin/products/new
	- [x] /admin/products/[handle]

### Does this introduce a breaking change? :warning:
NO

### Other information

Some progress 🧪

image

### CODE REVIEW

**Comment by flybayer** [comment]
Sweet!! Also make sure you add a test for the `products/ssr` page

**Comment by flybayer** [comment]
@Zeko369 hey, do you have a status update on this?

**Comment by Zeko369** [comment]
@flybayer Will have the rest of them done by tomorrow

**Comment by flybayer** [comment]
@Zeko369 is this done?

File: examples/store/app/admin/pages/admin/products/index.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import {useQuery, Link} from 'blitz'
 import getProducts from 'app/products/queries/getProducts'
 
 function ProductsList() {
-  const [products] = useQuery(getProducts)
+  const [products] = useQuery((args) => getProducts({...args, orderBy: {id: 'desc'}}))
```
Comment (flybayer) [inline examples/store/app/admin/pages/admin/products/index.tsx:6]:
```suggestion
  const [products] = useQuery(getProducts, {orderBy: {id: 'desc'}})
```

**Comment by kandros** [comment]
wondering if running `yarn install` in root will install cypress on every example, maybe worth moving it to `peerDependency`?

**Comment by flybayer** [comment]
If every example uses the same version, then it'll be hoisted and installed once

**Comment by Zeko369** [comment]
@flybayer for now this is done, I'll add it to the github CI in another PR, and also I'll upgrade this when we get seeds to drop the db and seed it before running the tests.

Also using something like `faker.js` is a nice improvement to my static test strings

**Comment by flybayer** [comment]
@Zeko369 ok cool, let's get the CI working soon.

And you have a conflict with yarn.lock

**Comment by flybayer** [comment]
Merged in #335 — thank you so much @Zeko369!! (I got it running in CI)

**Pull request closed without merging**